python/n.py

```diff
@@ -552,7 +548,27 @@ def type_check_command(self, tree):
 		command = tree.children[0]
 
 		if command.data == "imp":
-			self.imports.append(importlib.import_module(command.children[0]))
+			try:
+				imp = importlib.import_module(command.children[0])
+				self.imports.append(imp)
+				try:
+					getattr(imp, "_values")
+				except:
+					self.errors.append(TypeCheckError(command.children[0], "Library %s not compatable." % command.children[0]))
+			except:
+				self.errors.append(TypeCheckError(command.children[0], "Library %s not found to import." % command.children[0]))
+		elif command.data == "imported_command":
+			l, c, *args = command.children
+			library = self.find_import(l)
+			if library == None:
+				self.errors.append(TypeCheckError(l, "Library %s not found." % l))
+			else:
+				try:
+					if c not in library._values():
+						self.errors.append(TypeCheckError(c, "Command %s in %s not found." % c, l))
+				except:
+					pass
+			
```

Comment:
So I moved imported_command into `type_check_expr` so that imported commands can return values

Reply:
That's where you can find the warning about how it's not supported yet

Reply:
ok move that stuff over to there